src/component/drawer/Drawer.jsx

```diff
@@ -0,0 +1,48 @@
+import './drawer.css';
+import HomeOutlinedIcon from '@mui/icons-material/HomeOutlined';
+import LabelOutlinedIcon from '@mui/icons-material/LabelOutlined';
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined';
+import DeleteOutlineOutlinedIcon from '@mui/icons-material/DeleteOutlineOutlined';
+import FaceOutlinedIcon from '@mui/icons-material/FaceOutlined';
```

Comment:
MUI provides named imports as well, so you can use named imports to make your code cleaner

Reply:
read about it and changed.